Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Rematch2"
 uuid = "351a7294-9038-49b6-b9cf-e076b05af63f"
 authors = ["RelationalAI"]
-version = "0.2.4"
+version = "0.2.6"
```

Comment:
it looks like our currently released version is `0.2.6`, so `main` is a bit off - we'll need to make sure to bump the version correctly before the next release (but I wouldn't worry about that in any of these PR's as that's just playing whack-a-mole)